schedule.go

```
@@ -0,0 +1,300 @@
+	for _, slot := range slots {
+		timeStr := fmt.Sprintf("%02d:%02d", slot.hour, slot.minute)
+
+		// Build the full line and wrap it to terminal width, indenting wrapped lines
+		// Color the time and tree separately using lipgloss
+		prefix := timeStyle.Render(timeStr) + " " + treeStyle.Render("├─") + " "
+		// Visual width of prefix: "HH:MM ├─ " = 9 characters (ANSI codes have zero width)
+		const prefixVisualWidth = 9
+
+		// Determine terminal width; fallback to 80 if unavailable
+		width := 80
+		fd := uintptr(os.Stdout.Fd())
```

golangci-lint (2.12.2)

[error] unnecessary conversion

(unconvert)